### PR TITLE
Update japanese translation

### PR DESCRIPTION
**Pull request by TakehideMorimoto**
I made a Japanese translate here.

### CODE REVIEW

**Review by flaix — CHANGES_REQUESTED** [review]
Hi! This is awesome!

I found some issues in your files, so you may want to go over them again and make sure you have no duplicate keys, no incorrect keys and no empty definitions.

What is more, though, is a technical issue. Your changes are based on a rather old commit and then merged into your master. This way I cannot merge them. A merge request should always come from a separate branch, not from master. (And only contain the changes for one feature/topic.) So can you create a new branch on top of the current master of `gitblit/gitblit` and cherry-pick only the one commit 17da704 onto it? I do think that you will probably have to create a new merge request for it.

If you don't know how to do that, that is also not a problem. In that case you can also add a new commit to fix the issues and I will then create new commit from your commits for you.

Please be advised that several issues with language files have been fixed (see PR #1380), which also touched the Japanese properties file.

In any case, thank you very much for taking the time and improving the Japanase translation!

**Comment by TakehideMorimoto** [comment]
Sorry. I will go over some issues that you point out, and make sure keys.
And I create new branch and new merge request another day.

File: src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties [hunk 1]
```diff
@@ -1,454 +1,1543 @@
...
+#^gb\..*$
+#訳済行の検索
+#^gb\..*\\u.*$
+#未訳行の検索
+#^gb\.[^\\]+$
+# 全体  788  訳済  600  未訳  184
+
+#変換後、\\n を \n に置換のこと。
+#注意!!! #のコメントには、なんとビックリ数に限りがあるらしく、一定数を超えると起動しなくなってしまう。
+#        そうなった場合、優先度を考えてコメントを削ったり、複数行のコメントを1行にしたりして減らすこと。
```
[the remark this answers is not in the document]
Comment (flaix) [reply src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties:11]: Deleted

File: src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties [hunk 2]
```diff
@@ -1,454 +1,1543 @@
...
+gb.patchsetAlreadyMerged = This patchset has been merged into {0}.
+gb.patchsetNotMergeable = This patchset can not be automatically merged into {0}.
+gb.patchsetNotMergeableMore = This patchset must be rebased or manually merged into {0} to resolve conflicts.
+gb.patchsetNotApproved = This patchset revision has not been approved for merging into {0}.
+gb.patchsetNotApprovedMore = A reviewer must approve this patchset.
+gb.patchsetVetoedMore = A reviewer has vetoed this patchset.
+#書き込み
+gb.write = \u66f8\u304d\u8fbc\u307f
+#コメント
+gb.comment = \u30b3\u30e1\u30f3\u30c8
```
[the remark this answers is not in the document]
Comment (flaix) [reply src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties:1071]: Deleted

File: src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties [hunk 3]
```diff
@@ -1,454 +1,1543 @@
...
+gb.excludePermission = {0} (exclude)
+gb.viewPermission = {0} (view)
+gb.clonePermission = {0} (clone)
+gb.pushPermission = {0} (push)
+gb.createPermission = {0} (push, ref creation)
+gb.deletePermission = {0} (push, ref creation+deletion)
+gb.rewindPermission = {0} (push, ref creation+deletion+rewind)
+#権限
+gb.permission = \u6a29\u9650
+gb.regexPermission = this permission is set from regular expression \"{0}\"
```
[the remark this answers is not in the document]
Comment (flaix) [reply src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties:750]: Deleted

File: src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties [hunk 4]
```diff
@@ -1,454 +1,1543 @@
...
+#連結セット
+gb.federationSets = \u9023\u7d50\u30bb\u30c3\u30c8
+#メッセージ
+gb.message = \u30e1\u30c3\u30bb\u30fc\u30b8
+##gb.federation から gb.tokens (連結一覧) に入り、何れかの行の gb.sendProposal をクリックすると遷移する gb.sendProposal 画面にある。
+#この Gitblit 環境の URL
+#gb.myUrlDescription = the publicly accessible url for your Gitblit instance
+gb.myUrlDescription = \u3053\u306e Gitblit \u74b0\u5883\u306e URL
+#送信先
+#gb.destinationUrl = send to
```
[the remark this answers is not in the document]
Comment (flaix) [reply src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties:323]: All removed

File: src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties [hunk 5]
```diff
@@ -1,454 +1,1543 @@
...
+#環境設定
+gb.preferences = \u74b0\u5883\u8a2d\u5b9a
+#アカウント設定
+gb.accountPreferences = \u30a2\u30ab\u30a6\u30f3\u30c8\u8a2d\u5b9a
+#アカウントを設定します。
+gb.accountPreferencesDescription = \u30a2\u30ab\u30a6\u30f3\u30c8\u3092\u8a2d\u5b9a\u3057\u307e\u3059\u3002
+#言語設定
+gb.languagePreference = \u8a00\u8a9e\u8a2d\u5b9a
+#「言語設定」の解説。不要と思われる。
+gb.languagePreferenceDescription =
```
[the remark this answers is not in the document]
Comment (flaix) [reply src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties:1438]: Filled in with English default

File: src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties [hunk 6]
```diff
@@ -1,454 +1,1543 @@
...
+gb.emailMeOnMyTicketChanges = Email me on my ticket changes
+gb.emailMeOnMyTicketChangesDescription  = Send me an email notification for changes that I make to a ticket
+#「表示名」の解説。不要と思われる。
+gb.displayNameDescription =
```
[the remark this answers is not in the document]
Comment (flaix) [reply src/main/java/com/gitblit/wicket/GitBlitWebApp_ja.properties:1442]: Filled in with English default

**Comment by flaix** [comment]
I have fixed the submission and added it under a new commit to the main branch.
Closing this as done.

**Pull request closed without merging**